vagrant-libvirt.gemspec

```diff
@@ -20,9 +20,11 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rspec-expectations", "~> 2.14.0"
   gem.add_development_dependency "rspec-mocks", "~> 2.14.0"
 
-  gem.add_runtime_dependency 'fog-libvirt', '>= 0.3.0'
-  gem.add_runtime_dependency 'nokogiri', '~> 1.6.0'
   gem.add_runtime_dependency 'fog-core', '~> 1.43.0'
+  gem.add_runtime_dependency 'fog-libvirt', '>= 0.3.0'
+
+  # Make sure to use the same version of Nokogiri as Vagrant (https://git.io/v9sSO).
+  gem.add_runtime_dependency 'nokogiri', '~> 1.7.1'
```

Comment:
Someone suggested using '~ 1.6' to allow this to remain working on older versions as this should then use which ever version is required by vagrant.